build/AzurePipelinesTemplates/WindowsAppSDK-BuildAndTestMRT-Steps.yml

```diff
@@ -1,128 +1,121 @@
 
-# Common code for performing Binary Analysis.
-- ${{ if eq(parameters.RunSDLBinaryAnalysis, 'true') }}:
-  - template: WindowsAppSDK-BinaryAnalysis-steps.yml
-    parameters:
-      outputDirectory: 'BuildOutput'
-      # ToDo: Enable PREFast to run downstream from here, once the SDL template has been updated to invoke the 
-      # newer SDLNativeRules@3 task, and remove the explicit call to it above.
-      enablePREFast: false
+- task: PowerShell@2
+  name: BuildBinaries
+  inputs:
+    filePath: 'BuildAll.ps1'
+    arguments: -Platform "$(buildPlatform)" -Configuration "$(buildConfiguration)" -AzureBuildStep "BuildMRT"
 
 # component detection must happen *within* the build task
 - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
 
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish mrtcore binlog'
-  condition: succeededOrFailed()
-  inputs:
-    PathtoPublish: $(Build.SourcesDirectory)/BuildOutput/mrtcore.$(buildPlatform).$(buildConfiguration).binlog
-    artifactName: binlogs
-
-# Run the test locally on the Azure VM.
-# Note arm is currently not exercised.
-- task: VSTest@2
-  displayName: 'test MRT: ManagedTests'
-  condition: and(succeeded(), or(eq(variables['buildPlatform'], 'x86'), eq(variables['buildPlatform'], 'x64')))
-  inputs:
-    testSelector: 'testAssemblies'
-    testAssemblyVer2: '**\MrtCoreManagedTest.build.appxrecipe'
-    searchFolder: 'BuildOutput\$(buildConfiguration)\$(buildPlatform)\MrtCoreManagedTest'
-    testRunTitle: 'test MRT: ManagedTests - $(buildPlatform)'
-    platform: '$(buildPlatform)'
-    configuration: '$(buildConfiguration)'
-
-- task: VSTest@2
-  displayName: 'test MRT: UnitTests'
-  # These 3 MRT test suites were previously being run in Helix, but they are now being run via WindowsAppSDK-RunTestsInPipeline-Job.yml instead.
-  # Resort to running these tests right here on the current job agent if and only if we've got here from ProjectReunion-CI.yml, because we won't 
-  # be running these tests via WindowsAppSDK-RunTestsInPipeline-Job.yml later in this specific flow. Otherwise, we can generally skip these tests.
-  condition: and(and(succeeded(), or(eq(variables['buildPlatform'], 'x86'), eq(variables['buildPlatform'], 'x64'))), in(variables['Build.Reason'], 'IndividualCI', 'BatchCI'))
-  inputs:
-    testSelector: 'testAssemblies'
-    testAssemblyVer2: '**\MrmUnitTest.dll'
-    searchFolder: 'BuildOutput\$(buildConfiguration)\$(buildPlatform)\MrmUnitTest'
-    testRunTitle: 'test MRT: UnitTests - $(buildPlatform)'
-    platform: '$(buildPlatform)'
-    configuration: '$(buildConfiguration)'
+# TODO: Reenable Managed tests after https://microsoft.visualstudio.com/OS/_workitems/edit/44407037
```

Comment:
Looks like we only aim to bring back the Managed test here in the future, but not the other 3 TAEF tests?
Pre-OneBranch, those 3 TAEF tests were being run here for the PR validation flow, which does not run tests in the 1ES hosted pipeline environment (hence those 3 TAEF tests were being run here instead to make up for that).
In the OneBranch world, do we simply stop running those 3 TAEF tests during the PR validation flow, or we've enabled tests to run in the 1ES hosted pipeline environment in the flow?

Reply:
On develop, we are running these 3 other sets of MRTCore TAEF tests for our PR validation flow in the 1ES hosted pipeline environment.